test/unit/mockapi/mockapi.test.ts

```diff
@@ -12,7 +12,7 @@ describe("from file", () => {
     mockapi.mock.google.root
       .get({ search: /test.*/ })
       .reply({ status: 200, data: { message: "worked" } });
-    const { status, data } = await axios.get(
+    const { status, data, invalid } = await axios.get(
```

Comment:
not sure why we need this invalid? was this to test eslint failing instead of warning?

Reply:
Testing the change... we need to see red on CI to see if it worked:

<img width="1224" alt="image" src="https://github.com/kiegroup/act-js/assets/2906988/7b56e175-7ef7-49bb-b06d-1ebd09eac591">

Reply:
also this is pretty cool:

<img width="804" alt="image" src="https://github.com/kiegroup/act-js/assets/2906988/7b7c58cd-f86a-4e63-8b59-3d9099df7cff">

Reply:
I know!